pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -165,10 +164,23 @@ stdenv.mkDerivation rec {
   ++ optional (withNgspice) libngspice
   ++ optional (debug) valgrind;
 
+  # some ngspice tests attempt to write to $HOME/.cache/
+  XDG_CACHE_HOME = "$TMP";
+  # failing tests still attempt to create $HOME though
+  # and the CLI tests seem to also use $HOME...
+  HOME = "$TMP";
```

Comment:
Is it sufficient to set just `HOME`, given the comment about tests attempting to write to `$HOME` and `$HOME/.cache/`?

Reply:
not sure, running some test builds now

Reply:
`HOME` is indeed sufficient, thanks for the fresh eyes

and they haven't unified their use to `XDG_CACHE_HOME` in the mean time,
so that alone hasn't become sufficient again in the mean time
(i've submitted a question about their intentions around these)